setup-winston.js

```diff
@@ -0,0 +1,60 @@
+const { createLogger, format, transports } = require('winston');
```

Comment:
Line space please

Reply:
should't this be reflected in eslint rules?

Reply:
That's a good question. I was under the assumption that we added the relevant rules here already.

@v0vs, any idea where we have the relevant rules?

Reply:
Or are these still just part of our standard, but are not enforced?